### PR TITLE
[GH-5954] - Updated Authentication Page

### PR DESCRIPTION
**Pull request by adithyaakrishna**
Signed-off-by: Adithya Krishna <adikrish@redhat.com>

#### Summary

- Adds supported authentication methods to the introduction section of the docs site

#### Ticket Link

Fixes https://github.com/mattermost/docs/issues/5954

### CODE REVIEW

File: source/configure/authentication-configuration-settings.rst [hunk 1]
```diff
@@ -4,6 +4,13 @@ Authentication configuration settings
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
+Mattermost supports up to four distinct, concurrent methods of **Authentication**:
+
+- An OpenID provider
+- A SAML provider
+- An LDAP instance (ex Active Directory, OpenLDAP)
```
Comment (cwarnermm) [inline source/configure/authentication-configuration-settings.rst:11]:
```suggestion
- An LDAP instance (e.g., Active Directory, OpenLDAP)
```

Comment (cwarnermm) [reply source/configure/authentication-configuration-settings.rst:11]: Changed "ex" as example to "e.g.," to be consistent with existing documentation.

**Review by cwarnermm — COMMENTED** [review]
Thanks, @adithyaakrishna! I've provided one suggested update inline.

File: source/configure/authentication-configuration-settings.rst (hunk 1 above)
Comment (adithyaakrishna) [reply source/configure/authentication-configuration-settings.rst:11]: Done @cwarnermm :)

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**